rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/moderationrequest/Sw360ModerationRequestService.java

```diff
@@ -439,12 +447,41 @@ public ModerationState assignRequest(@NotNull ModerationRequest request, @NotNul
      * @return Count of open critical CRs
      * @throws TException Throws exception in case of errors.
      */
-    public Integer getOpenCriticalCrCountByGroup(String group) {
-        try {
-            return getThriftModerationClient().getOpenCriticalCrCountByGroup(group);
-        } catch (TException e) {
-            log.error("Error in getting open critical CR count by group: ", e);
-            return 0;
+	public Integer getOpenCriticalCrCountByGroup(String group) {
```

Comment:
Please reformat to use spaces for indent instead of tabs.

Reply:
Addressed.